pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "astro-tigger-lsm"
+version = "1.7.4"
+description = "Python libraries and command-line tools for manipulating Tigger LSMs"
+authors = ["Oleg Smirnov <osmirnov@gmail.com>"]
+readme = "README.rst"
+packages = [
+    {include = "Tigger"},
+]
+
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

Comment:
Include Python ^3.6 - this will likely break the test suite downstream - testing 3.6 -> 3.10

Reply:
Testing on a Python 3.6 machine doesn't make sense though. Especially since, the packaging tests are only done for 3.{8,9,10}. Python 3.6 was also released 7 years ago, it's too old for the more *comprehensive tests*